avro-fastserde/src/main/java/com/linkedin/avro/fastserde/FastSerdeBase.java

```diff
@@ -129,11 +129,11 @@ protected Class compileClass(final String className, Set<String> knownUsedFullyQ
        */
       compileResult = compiler.run(null, null, null, "-cp", compileClassPathForCurrentFile, filePath, "-XDuseUnsharedTable");
     } catch (Exception e) {
-      throw new FastSerdeGeneratorException("Unable to compile:" + className, e);
+      throw new FastSerdeGeneratorException("Unable to compile:" + className + ", and source file: " + filePath, e);
```

Comment:
nit: "unable to compile ... _FROM_ <path>" ?

Reply:
Will do.